sdk/driver/integration/suite_test.go

```diff
@@ -114,35 +112,10 @@ func (s *suite) doTestNativeParse(t *testing.T, filename string) {
 	r.Nil(err)
 
 	expected := getAST(r, filename)
-
-	EqualText(r, expected, NativeParseResponseToString(r, res))
-}
-
-func NativeParseResponseToString(r *require.Assertions, res *protocol.NativeParseResponse) string {
-	var s struct {
-		Status string      `json:"status"`
-		Errors []string    `json:"errors"`
-		AST    interface{} `json:"ast"`
-	}
-
-	s.Status = strings.ToLower(res.Status.String())
-	s.Errors = res.Errors
-	if len(s.Errors) == 0 {
-		s.Errors = make([]string, 0)
-	}
-
-	err := json.Unmarshal([]byte(res.AST), &s.AST)
-	r.Nil(err)
-
-	buf := bytes.NewBuffer(nil)
-	e := json.NewEncoder(buf)
-	e.SetIndent("", "    ")
-	e.SetEscapeHTML(false)
-
-	err = e.Encode(s)
+	resstr, err := NativeParseResponseToString(res)
```

Comment:
* `resstr` is probably not the best name for this variable
* `NativeParseResponseToString` should probably be just `String`, implementing the Stringer interface.

Reply:
See above.